src/Client.php

```diff
@@ -401,6 +402,33 @@ public function send($method, $payload, $path, $params = [], array $headers = []
             return null;
         }
 
+        $responseParsers = [
+            'application/json' => [$this, 'parseJson'],
+            'application/pdf' => [$this, 'parsePdf'],
+        ];
+
+        if (preg_match('/^(\w+\/\w+).*/i', $response->getHeaderLine('Content-Type'), $matches)) {
```

Comment:
> i (PCRE_CASELESS)
> If this modifier is set, letters in the pattern match both upper and lower case letters.

`/i` is used if you are searching for a pattern... instead above `\w` is used and it already matches lowercase and uppercase letters